### DISCUSSION TITLE
Patching the Boot partition

### DISCUSSION THREAD

**Discussion by Jofesine** [ntherning/StorytelReaderMods#6] [General]
Hello, i have some issues with the "patching the boot partition! part. I dont know what im doing wrong but this is the errors i get:

![Skärmavbild 2024-07-23 kl  00 49 30](https://github.com/user-attachments/assets/e4690868-40a9-41f3-a8d7-d41432e16322)

im working on a MAC, and i tried in both makrom mode and normal mode (and i have a 2nd gen)

Any tips?

**Comment by ntherning** [comment]
Hi, it looks like you have been trying the instructions for how to make your own patched `boot.img` file. You most likely don't need to do this. Instead please just follow the [Quickstart instructions](https://github.com/ntherning/StorytelReaderMods?tab=readme-ov-file#quickstart) at the start of the README.

**Reply by Jofesine** [reply]
Hey! Yes i completed the quickstart part, but everytime i start the reader i still only get to the ”storytel reader” app?

**Comment by ntherning** [comment]
I see. You need to use adb to connect to the device and install an alternative launcher app, e.g. RelaunchX. The  Recipes section in the README shows how to install and launch F-Droid via adb. Then you can find RelaunchX in the F-Droid app and install that. Then reboot the device and it should ask whether you want to launch storytel reader app or RelaunchX.

**Reply by Jofesine** [reply]
ahh! i made it! perfect! thanks for your help!
